modules/git-workflow-github.md

```diff
@@ -270,14 +273,19 @@ at (doing a code review), and approve a good solution.
   - Describe the targets in the PR body so that readers can understand
     in more details what the PR is about
   - If your PR resolves an issue just add `Resolves #3` somewhere 
-    in the comment (`3`
-    is a number of an issue you can see beside its title). GitHub will
-    bind your PR to that issue and the issue will get closed
+    in the comment (`3` is a number of an issue you can see beside its title). 
+    GitHub will bind your PR to that issue and the issue will get closed
     (as resolved) once your PR is merged.
 - Draw attention
   - PR created from a fork is bound to the original repo by GitHub,
     so navigate to the original repo, Pull Requests and open a PR
-    you've opened
+    you've opened  
+```
+    You may not assign reviewers if you're not an owner of the repo
+    or not a part of organization. Try identifying the owner or repo
+    collaborators and mention them in a comment by their names.
+    If unable to identify relevant persons then just skip this step.
+```
   - There is a PR management section on the right side of the PR.
     Click **Reviewers** and assign two or three of your peers.
```

Comment:
Perhaps line numbering changed after some other additions. So Line 282 became something different. The idea is to have note on inability to add reviewers after the suggestion to do so, i.e. after "... and assign two or three of your peers."

Reply:
I just started by adding a link and probably because of this I inserted a note in the wrong place. Yes, I tried to make a block quote and suggested this option as 'plan B', it fits within the meaning. I've done new changes, @OleksiyRudenko please check if everything alright.

p.s: I also made a change with combining the text, otherwise it was a break: "somewhere in the comment (3 is a number" in the commit 7635a631eb36eaf8738ea1bcabcbd0f2c2f9d4a1